packages/components/src/Select.tsx

```diff
@@ -259,7 +259,25 @@ export const GroupedSelect = <
   return <ParameterizedBaseSelect findValue={findValue} value={value} options={options} {...props} />;
 };
 
-export const AsyncCreatableSelect = (props: any) => {
+export type GroupBase<Option> = RSelectGroupBase<Option>;
```

Comment:
I need some help understanding why it's necessary to redefine `RSelectGroupBase`, and to understand the usage below at line 267.

Reply:
This is vestigial from my first pass at the generic types in `WorkspaceTagSelect`, I'll take it out.